examples/tools/cmake/cmake_toolchain/local_flow.rst

```diff
@@ -0,0 +1,83 @@
+CMakeToolchain in the developer flow
+====================================
+
+.. note::
+
+    You can find the following example in this repository
+
+    .. code-block:: bash
+
+        $ git clone https://github.com/conan-io/examples2.git
```

Comment:
I'm not convinced about the idea that we make people clone a repository and cd to an empty folder, maybe we can skip this step for these cases but leave the test in the examples2 repository. Or just put the results of the `conan new` there and explaining in the example that it's the product of running `conan new...`

Reply:
I agree, I think this is "core" enough to be in the main thread of the tutorial, not as a side example.